plugin/evm/mempool.go

```diff
@@ -259,16 +274,53 @@ func (m *Mempool) addTx(tx *Tx, force bool) error {
 	for utxoID := range utxoSet {
 		m.utxoSpenders[utxoID] = tx
 	}
+
+	m.bloom.Add(&GossipAtomicTx{Tx: tx})
+	reset, err := gossip.ResetBloomFilterIfNeeded(m.bloom, txGossipBloomMaxFilledRatio)
```

Comment:
Do we have a metric for resets?

Reply:
We don't have metrics on this... but we should have metrics on all of the new gossip stuff and this is definitely a metric that we would want to track